lib/iris/tests/unit/util/test_mask_cube.py

```diff
@@ -29,24 +26,27 @@ def full2d_global():
 
 
 class MaskCubeMixin:
-    def assertOriginalMetadata(self, cube, func):
+    def assertOriginalMetadata(self, request, cube, func):
```

Comment:
Should probably change the syntax of this name as it currently looks like unittest